dist/index.html

```diff
@@ -3,18 +3,28 @@
 <head>
   <meta charset=utf-8>
   <html lang="en" dir="ltr">
-  <link rel="stylesheet" href="./styles.css">
+  <link rel="stylesheet" src="src/css/styles.css" />
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
   <title>Travel Tracker</title>
 </head>
 <body>
-  <header class="top-header">
+  <section class="login-section" id="loginSection">
+    <div class="login-inputs">
+      <label for="username">Username:</label>
+      <input type="text" id="username" name="username">
+      <label for="pass">Password:<label>
+      <input type="password" id="pass" name="password" class="password"required>
```

Comment:
I think the for class in the label should be "password" to match the input's name class.